mod/consensus/pkg/cometbft/service/abci.go

```diff
@@ -394,7 +378,7 @@ func (s *Service[_]) validateFinalizeBlockHeight(
 	req *cmtabci.FinalizeBlockRequest,
 ) error {
 	if req.Height < 1 {
-		return fmt.Errorf("invalid height: %d", req.Height)
+		return fmt.Errorf("FinalizeBlock: %w %v", errInvalidHeight, req.Height)
 	}
```

Comment:
Looks good, nit: error message generally starts with smaller case letter. Similar comments at few other places.

Reply:
Thanks for spotting this, @nidhi-singh02 . Fixed